src/main/java/com/iexec/core/contribution/ContributionService.java

```diff
@@ -18,33 +18,27 @@
 
 import com.iexec.common.replicate.ReplicateStatus;
 import com.iexec.core.replicate.Replicate;
-import com.iexec.core.replicate.ReplicatesService;
 import org.springframework.stereotype.Service;
 
 import java.util.HashSet;
+import java.util.List;
 import java.util.Optional;
 import java.util.Set;
 
 @Service
```

Comment:
I don't think we need a service anymore, a ContributionHelper is enought.

Reply:
Same for `PredictionService`.